plugins/language_nim.lua

```diff
@@ -88,14 +88,37 @@ for _, type in ipairs(standard_generic_types) do
   table.insert(patterns, { pattern = type.." +%f[%w]", type = "keyword2" })
 end
 
+local function string_pattern(start, stop, syntax)
+  return {
+    pattern = { start, stop, '\\' },
+    type    = "string",
+    syntax  = syntax and {
+      patterns = syntax,
+      symbols  = {},
+    } or nil
+  }
+end
+
+local interpolation_syntax = {
+  { pattern = { '{{', '}}' }, type = "string" },
+  { pattern = { '{', '}', '\\' }, type = "keyword2", syntax = { patterns = patterns, symbols = symbols, } },
```

Comment:
Would this be equivalent to:
```suggestion
  { pattern = { '{', '}', '\\' }, type = "keyword2", syntax = ".nim" },
```